tests/unit/server/bundle_manager/__init__.py

```diff
@@ -0,0 +1,234 @@
+import unittest
```

Comment:
These bundle manager tests shouldn't be in this PR, right?

Reply:
Forgot to mention this in the PR, but this PR is based on #2795, as I used some of the common utilities from that PR for testing here. It would be good to merge this once #2795 is merged.